packages/xo-server-perf-alert/src/index.js

```diff
@@ -524,11 +524,45 @@ ${monitorBodies.join('\n')}`
                   })
                 }
 
-                result.listItem = `  * ${result.objectLink}: ${
-                  result.value === undefined
-                    ? "**Can't read performance counters**"
-                    : result.value.toFixed(1) + typeFunction.unit
-                }\n`
+                const isManagementAgentDetected = (vm, guestMetrics) => {
+                  if (
+                    (vm.power_state !== 'Running' && vm.power_state !== 'Paused') ||
+                    vm.metrics === undefined ||
+                    guestMetrics === undefined
+                  ) {
+                    return
+                  }
+
+                  const { major, minor } = guestMetrics.PV_drivers_version
+                  const hasPvVersion = major !== undefined && minor !== undefined
+                  const pvDriversDetected = guestMetrics.PV_drivers_detected ?? hasPvVersion
+
+                  return pvDriversDetected || hasPvVersion || guestMetrics.other['feature-static-ip-setting'] === '1'
+                }
+
+                const getListItem = () => {
+                  if (result.value == null) {
+                    return "**Can't read performance counters**"
+                  }
+
+                  // Check the presence of the guest tools in the monitored object and ensure the correct report of the memory usage
```

Comment:
Ok but why do we need to check that for `'memoryUsage'` specifically? I think it was because RRDs are filled with `0`'s when there are no guest tools to report the actual values?